src/main/java/com/sonyericsson/jenkins/plugins/bfa/statistics/StatisticsLogger.java

```diff
@@ -128,12 +129,19 @@ public void run() {
             if (build instanceof AbstractBuild) {
                 AbstractBuild abstractBuild = (AbstractBuild)build;
                 Node node = abstractBuild.getBuiltOn();
-                nodeName = node.getNodeName();
+                if (node != null) {
+                    nodeName = node.getNodeName();
```

Comment:
Probably you just want to fail the run, but OK

Reply:
It is set to unknown above.